hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -1512,6 +1515,7 @@ String initializeUserAgent(final AbfsConfiguration abfsConfiguration,
       sb.append(HUNDRED_CONTINUE);
       sb.append(SEMICOLON);
     }
+    sb.append(" ").append(abfsConfiguration.getPreferredHttpOperationType()).append(";");
```

Comment:
empty string and semicolon should be constants

Reply:
taken.